flutter_mobx/lib/version.dart

```diff
@@ -1,4 +1,4 @@
 // Generated via set_version.dart. !!!DO NOT MODIFY BY HAND!!!
 
 /// The current version as per `pubspec.yaml`.
-const version = '2.0.6+5';
```

Comment:
looks like this version was lagging behind and got updated once the set_version was run. Could you please include a changelog entry for this by reading the changes done in the past version.